### PR TITLE
add missing prop-types to question audit

### PR DESCRIPTION
**Pull request by alxnddr**
Added propTypes I missed for https://github.com/metabase/metabase/pull/16138

How to test:
- Run the app in EE mode
- Ensure there are no related errors for http://localhost:3000/admin/audit/questions/all and http://localhost:3000/admin/audit/question/*question_id*/activity (but there will be a bunch of other errors :()

### CODE REVIEW

File: enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditParameters.jsx [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
+      placeholder: PropTypes.string.isRequired,
+    }),
+  ),
+  children: PropTypes.func,
 };
 
 export default class AuditParameters extends React.Component {
-  props: Props;
-  state: State;
+  static propTypes = propTypes;
```
Comment (nemanjaglumac) [inline enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditParameters.jsx:19]: I thought we agreed not to use static propTypes?

Comment (alxnddr) [reply enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditParameters.jsx]: Oh, fixed that

File: enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx [hunk 2]
```diff
@@ -20,6 +20,19 @@ import cx from "classnames";
 
 const getColumnName = column => column.remapped_to || column.name;
 
+const propTypes = {
+  series: PropTypes.array,
+  visualizationIsClickable: PropTypes.func,
```
Comment (nemanjaglumac) [inline enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx:25]: Not knowing anything about this prop (judging purely from its name), it seems like it should be boolean, no?

**Comment by nemanjaglumac** [comment]
Can you please add a bit more details in the PR description on how to validate this manually? Which page (part of the app) contains these components?

File: enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx (hunk 2 above)
Comment (alxnddr) [reply enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx:25]: I thought so too, but it is used as a function

**Comment by alxnddr** [comment]
@nemanjaglumac added that

**Review by nemanjaglumac — APPROVED** [review]

File: enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx (hunk 2 above)
Comment (nemanjaglumac) [reply enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx:25]: 🤷‍♂️

**Pull request merged**